server/controllers/StatusController.js

```diff
@@ -39,8 +41,17 @@ const getStatus = async(req, res) => {
     case 'in_progress': {
       const mission = latestMission;
       const vehicle = await getVehicle(latestMission.vehicle_id);
-      vehicles = [vehicle];
       const status = 'in_mission';
```

Comment:
@TalAter I didn't want to change the general `status` being returned each time because the `vehicle` and `mission` objects already have the more detailed stages/statuses.

Reply:
👍